app/controllers/api/functions.php

```diff
@@ -474,6 +482,42 @@
         ]), Response::MODEL_RUNTIME_LIST);
     });
 
+App::get('/v1/functions/specifications')
+    ->groups(['api', 'functions'])
+    ->desc('Get available function runtime specifications')
+    ->label('scope', 'functions.read')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'functions')
+    ->label('sdk.method', 'getSpecifications')
```

Comment:
This should be `listSpecifications` for consistency, same for the description

Reply:
Updated 👍